commercetools-models/src/main/java/io/sphere/sdk/carts/LineItemDraft.java

```diff
@@ -31,11 +32,20 @@
 
     Integer getVariantId();
 
+    /**
+     * Possible custom tax rate.
+     * If set, this tax rate will override the tax rate selected by the platform.
```

Comment:
Now the external tax rate does not override. External tax rates need to be set when the cart has the taxMode External.